hip-service/Link/Patient/Communication.cs

```diff
@@ -5,8 +5,8 @@ namespace hip_service.OTP
 {
     public class Communication
     {
-        public CommunicationMode Mode { get; set; }
-        public string Value { get; set; }
+        private readonly CommunicationMode Mode;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field 'Mode' or refactor the code to use its value.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4832436)

Reply:
Ignore this.